main/options/options.cc

```diff
@@ -351,7 +351,14 @@ buildOptions(const vector<pipeline::semantic_extension::SemanticExtensionProvide
     options.add_options("advanced")("enable-experimental-lsp-signature-help",
                                     "Enable experimental LSP feature: Signature Help");
     options.add_options("advanced")("enable-experimental-lsp-quick-fix", "Enable experimental LSP feature: Quick Fix");
-    options.add_options("advanced")("enable-all-experimental-lsp-features", "Enable every experimental LSP feature.");
+    options.add_options("advanced")(
+        "enable-all-experimental-lsp-features",
+        "Enable every experimental LSP feature "
+        "(WARNING: Experimental features may be destabilizing/crashing; not recommended for non-developers. "
+        "Users wishing to beta-test LSP features should consider `--enable-all-beta-lsp-features`, instead)");
+    options.add_options("advanced")(
+        "enable-all-beta-lsp-features",
+        "Enable all beta LSP features (suitable for end users wishing to beta-test new LSP features)");
```

Comment:
I think that this will probably make it harder to get people's individual feedback (i.e., if one beta feature interferes with another feature). I'll probably still opt to have people turn on my beta features individually, but ¯\_(ツ)_/¯ up to you how you want to use this option.

Reply:
To echo an internal conversation, this change steers us to separate these different pre-release phases:

* *experimental* (alpha testing): Early feature exploration, high-touch relationships with a select few users, where the goal is to collect design/usability feedback.
* *beta* (canarying): Late feature hardening, low-touch interactions with a larger group of users, where the goal is to collect stability feedback before releasing to the general population.

My current feature has progressed to the latter stage, so my intent/hope was to simplify life for that population of beta-testing users, not to impact users of features in the experimental stage.

Reply:
FWIW, I'll hold off on shipping this for now to make sure I'm not underestimating/underappreciating the impact it'll have on your process.